File: src/main/java/com/clinica/atendimento/repository/TerapiaConvenioRepository.java
package com.clinica.atendimento.repository;

import com.clinica.atendimento.model.TerapiaConvenio;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface TerapiaConvenioRepository extends JpaRepository<TerapiaConvenio, Long> {
}

File: src/main/kotlin/com/clinica/atendimento/repository/TerapiaConvenioRepository.kt
package com.clinica.atendimento.repository

import com.clinica.atendimento.model.TerapiaConvenio
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface TerapiaConvenioRepository : JpaRepository<TerapiaConvenio, Long>